app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -1,37 +1,55 @@
 package io.pumpkinz.pumpkinreader.model;
 
 import org.parceler.Parcel;
-
 import java.io.Serializable;
-import java.util.Date;
+import java.util.List;
 
 
 @Parcel
-public class Comment implements Serializable {
+public class Comment extends Item implements Serializable {
+
+    int parent;
+    List<Integer> kids;
+    Comment parentComment;
+    List<Comment> comments;
```

Comment:
Yes, but eventually we will need to retrieve it and store it in an object.
Or do you think we should store it somewhere else?